### ISSUE TITLE
Typo in Readme

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#78]
WP Acessibility Facebook Group

should be Accessibility
<img width="627" alt="links at the top of the readme on WordPress dot org" src="https://user-images.githubusercontent.com/13153456/204900134-b344172b-fc11-4de6-8101-1802fc4e464d.png">

**Comment by amberhinds** [comment]
This was fixed in 1.3.4
